app.rb

```diff
@@ -0,0 +1,131 @@
+require './teacher'
+require './student'
+require './book'
```

Comment:
**Optional**
- As all of the required files are on the same dir, it'd make more sense if you used `require_relative`